File: WebApp/WebApp/Models/Radnik.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace WebApp.Models
{
    public class Radnik
    {
        [Key]
        public int IdRadnik { get; set; }
        [ForeignKey("Nadredjen")]
        public int NadredjenId { get; set; }
        public Nadredjen Nadredjen { get; set; }
        [ForeignKey("Pozicija")]
        public int PozicijaId { get; set; }
        public Pozicija Pozicija { get; set; }
        public string Ime { get; set; }
        public string Prezime { get; set; }
        public DateTime PreiodRaspodele { get; set; }
    }
}

File: angular-app/src/app/models/radnik.ts
import { Bonus } from "./bonus";
import { Odbitak } from "./odbitak";

export class Radnik{
    IdRadnik: number;
    NadredjenId: number;
    PozicijaId: number;
    Ime: string;
    Prezime: string;
    PreiodRaspodele: Date;
}

export class RadnikModel{
    IdRadnik: number;
    Ime: string;
    Prezime: string;
    ImeNadredjenog: string;
    PrezimeNadredjenog: string;
    NazivPozicije: string;
    PreiodRaspodele: Date;
    IznosPlate: number;
    BonusiAktivni: Bonus[];
    OdbitciAktivni: Odbitak[];
    BrojDanaGodisnjegOdmora: number;
}